test/contracts/DynamicStringLengthCheck.sol

```diff
@@ -0,0 +1,7 @@
+pragma solidity ^0.4.2;
+
+contract DynamicStringLengthCheck {
+  function stringSetting(string oldText, string newText) pure public {
+    require(keccak256(abi.encodePacked(oldText)) == keccak256(abi.encodePacked(newText)));
```

Comment:
I was thinking that ideally this would be in the same contract above. I'm not sure why this would need to be in a separate contract?